Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.4.1-alpine
+
+ENV RUNTIME_PACKAGES="tini" \
```

Comment:
Addressed in https://github.com/Intellection/docker-gemstash/pull/1/commits/1e5fdc2dae3866706a2e5a527fcbb7161dd90178.